provd/proto/accessibility.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package accessibility;
+
+service AccessibilityDirectoryService {
+    rpc GetHighContract(GetHighConstrastRequest) returns (GetHighConstrastResponse) {}
+    rpc SetHighContract(SetHighConstrastRequest) returns (SetHighConstrastResponse) {}
+    rpc GetReducedMotion(GetReducedMotionRequest) returns (GetReducedMotionResponse) {}
+    rpc SetReducedMotion(SetReducedMotionRequest) returns (SetReducedMotionResponse) {}
+    rpc GetScreenReader(GetScreenReaderRequest) returns (GetScreenReaderResponse) {}
+    rpc SetScreenReader(SetScreenReaderRequest) returns (SetScreenReaderResponse) {}
+    rpc GetLargeText(GetLargeTextRequest) returns (GetLargeTextResponse) {}
+    rpc SetLargeText(SetLargeTextRequest) returns (SetLargeTextResponse) {}
+}
+
+message GetHighConstrastRequest {
+}
+
+message GetHighConstrastResponse {
+    bool success = 1;
+    string error = 2;
+    bool enabled = 3;
```

Comment:
In general, do we want to move forward with every call having a response with `success` and any error `messages`?